.gitignore

```diff
@@ -198,3 +198,6 @@ pip-selfcheck.json
 node_modules/
 bundle/
 webpack-stats.json
+
+# VS code
```

Comment:
Comment can be removed. It's already pretty clear from the folder name ;)

Reply:
fixed!